backend/app/api/endpoints/adapter/chat.py

```
@@ -1730,3 +1730,77 @@ async def get_search_engines(
+
+
+class ParseUrlsResponse(BaseModel):
+    """Response body for parsed URLs."""
+
+    results: list[ParsedUrlResponse]
+
+
+@router.post("/parse-urls", response_model=ParseUrlsResponse)
+async def parse_urls(
+    request: ParseUrlsRequest,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.10)

Unused function argument: `current_user`

(ARG001)